src/features/userProfile/UserProfile.tsx

```diff
@@ -96,12 +95,13 @@ const UserProfile: React.FC<UserProfileProps> = ({ path = '/' }) => {
       numberDecorator: paginatedDatasetResults.pageInfo.resultCount,
       selected: path === '/'
     },
-    {
-      name: 'Following',
-      link: `/${usernameParam}/following`,
-      numberDecorator: paginatedFollowingResults.pageInfo.resultCount,
-      selected: path === '/following'
-    }
+    // TODO (boandriy): Hiding temporarily to match new design
+    // {
+    //   name: 'Following',
+    //   link: `/${usernameParam}/following`,
+    //   numberDecorator: paginatedFollowingResults.pageInfo.resultCount,
+    //   selected: path === '/following'
+    // }
```

Comment:
Can you write in the TODO when it should be commented out, or remove the commented code